ae.html

```diff
@@ -1,9 +1,17 @@
 <!DOCTYPE html>
 <html>
 <head>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 <title>AUTOMOTIVE_ELECTRONICS</title>
+<link rel="stylesheet" href="ae.css">
 </head>
 <body>
-<p>AUTOMOTIVE ELECTRONICS</p>
+<h2>AUTOMOTIVE_ELECTRONICS</h2>
+<div class="btn">
+<button onclick="ae();"><i class="fa fa-plus-square"></i> Write here</button>

```

Comment:
Instead of `onClick` try adding an id to this button and then add an `event listener` in the javascript file.